lib/generator.rb

```diff
@@ -35,13 +37,43 @@ def test_cases
     cases.call(data)
   end
 
+  def metadata_repository_missing_message
+    <<EOM
+
+'#{METADATA_REPOSITORY}' repository not found.
+Try running the command:
+  git clone https://github.com/exercism/#{METADATA_REPOSITORY}.git ../#{METADATA_REPOSITORY}
+
+EOM
+  end
+
   def generate
+    check_metadata_repository_exists
+    generate_test_file
+    increment_version
+    increment_version_in_example
+  end
```

Comment:
I asked about maybe giving a choice to inhibit version increment with an environment variable.  It might look something like this:

```ruby
  def generate
    check_metadata_repository_exists
    generate_test_file
    unless ENV['NOINCREMENT']
      increment_version 
      increment_version_in_example
    end
  end
```